benefits/core/admin.py

```diff
@@ -151,3 +152,11 @@ def add_staff_user_to_group(user, request):
     if user.email in settings.GOOGLE_SSO_STAFF_LIST:
         staff_group = Group.objects.get(name=settings.STAFF_GROUP_NAME)
         user.groups.add(staff_group)
```

Comment:
I guess it's the same thing here...

Reply:
Done in https://github.com/cal-itp/benefits/pull/2325/commits/04d3c194d80ae4be4afffd92fefeef986410e93a